torch/fx/experimental/proxy_tensor.py

```diff
@@ -927,7 +944,7 @@ def wrap_key(f: Callable[_P, R], tensors: _P.args, tracer: _ProxyTracer, pre_dis
     def wrapped(*proxies: _P.args, **_unused: _P.kwargs) -> R:
         flat_proxies, proxies_spec = pytree.tree_flatten(proxies)
         assert len(flat_proxies) == len(flat_tensors)
-        with disable_proxy_modes_tracing() as m:
+        with disable_proxy_modes_tracing() as m, m.sym_mode.enable(False):
```

Comment:
This is important to avoid spewing unnecessary size accesses when we're initially tracking inputs. Previously they didn't show up because they were implicitly DCE'd